samples/cpp/facedetect.cpp

```diff
@@ -122,36 +114,27 @@ int main( int argc, const char** argv )
         if(image.empty()) cout << "Couldn't read ../data/lena.jpg" << endl;
     }
 
-    cvNamedWindow( "result", 1 );
+    namedWindow( "result", 1 );
```

Comment:
I would initiate windows at the beginning of the file and add a comment why you need this. Actually it is not really necessary I think.